fix(three node example): fixes three node example (#436)
interledger-rs/interledger-rs: examples/eth-xrp-three-nodes/README.md

By default, the XRP settlement engine generates new testnet XRPL accounts prefunded with 1,000 testnet XRP (a new account is generated each run). Alternatively, you may supply an `XRP_SECRET` environment variable by generating your own testnet credentials from the [official faucet](https://xrpl.org/xrp-test-net-faucet.html).

> The engines are part of a [separate repository](https://github.com/interledger-rs/settlement-engines) so you have to clone and install them according to [the instructions](https://github.com/interledger-rs/settlement-engines/blob/master/README.md). In case you've never cloned `settlement-engine`, clone it first.

Reviewer: ```suggestion
The engines are part of a [separate repository](https://github.com/interledger-rs/settlement-engines) so you have to clone and install them according to [the instructions in settlement-engine](https://github.com/interledger-rs/settlement-engines/blob/master/README.md). In case you've never cloned `settlement-engine`, the first step would be to clone the repository.
```

        interledgerjs/settlement-xrp
else
    which ilp-settlement-xrp &> /dev/null || error_and_exit "You need to install \"ilp-settlement-xrp\"."
    pushd "${SETTLEMENT_ENGINE_INSTALLL_DIR}" &>/dev/null
    if [ ! -e "settlement-engines" ]; then
        git clone https://github.com/interledger-rs/settlement-engines
    fi
    pushd settlement-engines &>/dev/null
-->

```bash #

> # Do this somewhere OUTER the interledger-rs directory otherwise you'll get an error.

Reviewer 2: ```suggestion
# This should be done outside of the interledger-rs ditrectory, otherwise it will cause an error
```

> # This should be done outside of the interledger-rs ditrectory, otherwise it will cause an error

Reviewer: ```suggestion
# This should be done outside of the interledger-rs directory, otherwise it will cause an error
```

--settlement_api_bind_address 127.0.0.1:3001 \
&> logs/node-bob-settlement-engine-eth.log &

> # Now go back to interledger-rs directory.

Reviewer: ```suggestion
# Now go back to the `interledger-rs` directory.
```

ILP_REDIS_URL=redis://127.0.0.1:6381/ \
ILP_HTTP_BIND_ADDRESS=127.0.0.1:8770 \
ILP_SETTLEMENT_API_BIND_ADDRESS=127.0.0.1:8771 \
cargo run --bin ilp-node &> logs/node-bob.log &

> # Start Charlie's node

Author: This is kind of confusing. This should be documented or fixed in the future.

Reviewer: According to the diagram and the way these nodes are available, we could even call charlie's node address as `example.charlie` to be consistent with `alice` and `bob`

Author: I suppose the reason is that if we set `Charlie`'s node as a peer, it would take 20secs or so to completely establish the route from Alice to Charlie. That looks inappropriate in this example. So we have to set this `child/parent` relation
